ckanext/stadtzhtheme/plugin.py

```diff
@@ -15,6 +15,8 @@
 from ckan import model
 from ckan.common import _
 
+from ckanext.stadtzhtheme import logic as l
```

Comment:
that is one of the conventions that I don't really agree with.. can we call this logic, please, instead of just `l`? 😅